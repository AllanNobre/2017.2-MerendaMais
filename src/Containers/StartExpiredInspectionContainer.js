import { connect } from 'react-redux';
import StartExpiredInspection from '../screens/startInspection/StartExpiredInspection';

const mapStateToProps = state => ({
  counselor: state.counselor,
  listOfExpiredScheduleInAGroup: state.list.listOfExpiredScheduleInAGroup,
});

const StartPendingInspectionContainer = connect(mapStateToProps)(StartExpiredInspection);

export default StartPendingInspectionContainer;
