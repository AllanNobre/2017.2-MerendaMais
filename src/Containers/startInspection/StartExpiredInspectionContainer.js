import { connect } from 'react-redux';
import StartExpiredInspection from '../../screens/startInspection/StartExpiredInspection';

const mapStateToProps = state => (
  {
    isLoading: state.application.isLoading,
    counselor: state.counselor,
    listOfExpiredScheduleInAGroup: state.list.listOfExpiredScheduleInAGroup,
  }
);

const StartExpiredInspectionContainer = connect(mapStateToProps)(StartExpiredInspection);

export default StartExpiredInspectionContainer;
