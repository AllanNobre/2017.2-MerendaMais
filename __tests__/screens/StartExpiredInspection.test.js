import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import configureStore from 'redux-mock-store';
// import configureStore from 'redux-mock-store';
import StartExpiredInspectionContainer from '../../src/Containers/StartExpiredInspectionContainer';

Enzyme.configure({ adapter: new Adapter() });

const mockStore = configureStore();

const initialState = {

  list: {
    listOfExpiredScheduleInAGroup: {
      codSchool: '',
      date: '',
      time: '',
    },
  },
};

const store = mockStore(initialState);

describe('Testing StartPendingInspection Screen', () => {
  it('renders as expected', () => {
    const wrapper = shallow(
      <StartExpiredInspectionContainer />,
      { context: { store } },
    );
    expect(wrapper).toMatchSnapshot();
  });
});
