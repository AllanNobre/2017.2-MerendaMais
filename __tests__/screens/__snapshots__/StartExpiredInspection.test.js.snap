// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing StartPendingInspection Screen renders as expected 1`] = `
<StartExpiredInspection
  dispatch={[Function]}
  listOfExpiredScheduleInAGroup={
    Object {
      "codSchool": "",
      "date": "",
      "time": "",
    }
  }
/>
`;
